### PR TITLE
Add back some rsync logging

### LINKED ISSUE #287: Too many files for deploy to list them all
https://github.com/whatwg/whatwg.org/pull/285 added 103k files which broke deploy on Travis due to exeeding the maximum log length. We have in total 108k files in the repo now.

https://github.com/whatwg/whatwg.org/pull/286 didn't work, and to get things working ASAP I'm going to suggest not logging the files at all.

Filing this issue to keep track of the problem.

### PR DESCRIPTION
**Pull request by annevk**
This also adds --stats for minimal logging in the CI log.

Closes #287 hopefully.

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -58,5 +58,7 @@ else
     # --verbose isn't used because there are too many files to list them all
     # without exceeding log size limits:
     # https://github.com/whatwg/whatwg.org/issues/287
-    rsync --archive --chmod="D755,F644" --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" ./whatwg.org ./*.whatwg.org "deploy@$SERVER:/var/www/"
+    rsync --archive --chmod="D755,F644" --compress --delete --stats --log-file="rsync-log.txt" --rsh="ssh -o UserKnownHostsFile=known_hosts" ./whatwg.org ./*.whatwg.org "deploy@$SERVER:/var/www/"
+    scp rsync-log.txt "deploy@$SERVER:/var/www/whatwg.org"
```
Comment (annevk) [inline deploy.sh]:
```suggestion
    scp rsync-log.txt "deploy@$SERVER:/var/www/whatwg.org/"
```

Comment (foolip) [reply deploy.sh]: Does it work without this suggestion, I guess not?

File: deploy.sh [hunk 2]
```diff
@@ -58,5 +58,7 @@ else
...
+    echo "Full rsync log at https://whatwg.org/rsync-log.txt"
```
Comment (foolip) [inline deploy.sh]: Oh, well that works :)

**Comment by annevk** [comment]
I think it works without because whatwg.org exists, but if it didn't it would rename I think rather than throw an error.

**Comment by annevk** [comment]
One worry I have is that the log file on the server will first be send back due to rsync before being overwritten. But that's probably okay?

**Comment by foolip** [comment]
The remote rsync-log.txt should be deleted by the first command because there won't be a whatwg.org/rsync-log.txt file. That seems harmless.

**Comment by annevk** [comment]
Oh my. The scp doesn't know about the known hosts.

**Pull request merged**